explorer.js

```diff
@@ -707,7 +717,8 @@ function ViewController($scope, SharedService) {
 		$scope.view.keys_selected = [];
 
 		$tb.DataTable().rows().data().each((data) => {
-			const link = document.querySelector(`[data-s3Key="${data.Key}"]`);
+			const etag = data.ETag.split('"').join("")
```

Comment:
isn't a `replace` more straightforward than split and join ?

Reply:
> isn't a `replace` more straightforward than split and join ?

Yes this might be better, fixed